build.gradle

```diff
@@ -446,3 +446,13 @@ gradleEnterprise {
         }
     }
 }
+
+allprojects {
+    normalization {
+        runtimeClasspath {
+            metaInf {
+                ignoreManifest()
```

Comment:
Just to be clear, this will ignore all `META-INF/MANIFEST.MF` files in all projects. Generally, we suggest to be as precise as possible, ignoring the least stuff. For instance, in this case I think you should get the same result just by adding `ignoreAttribute("Created-By")`.

Reply:
Good point 👍    Especially since we want to make sure `Multi-Release` is present for the core jar I think your suggestion makes sense